app/models/moderator_application.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+# Represents a moderator application.
+class ModeratorApplication < ApplicationRecord
+  LENGTH_RESTRICTION = {
+    maximum: 2000,
+    too_long: 'This is not the place for your life story. Limit reponses to 2000 charcters.'
+  }.freeze
+
+  belongs_to :user
+
+  # The ID of the user who is applying.
+  # Users may only have one moderator application each; however, they may edit it.
+  validates :user_id, presence: true, uniqueness: true
+
+  # The user must confirm that they are at least 18 years old.
+  # This data is not stored in the database, but the box must be checked on the form.
+  validates :at_least_18_years_old,
+            acceptance: { message: 'You must be at least 18 years old to apply to be a moderator.' }
```

Comment:
I ended up messing with the error messages in a weird way because the default error rendering didn't make any sense because the symbols are named poorly (along the lines of "`:why_mod` must be present", which doesn't make sense, because what is `:why_mod`?)

I feel like the real issue here is with my symbol names, but I'm not sure if it would be a good idea to name my fields `:why_does_the_applicant_want_to_be_a_moderator`, `:past_experience_as_a_moderator`, etc.. This is especially true because names should be to some extent self-documenting, and as is it probably makes even *less* sense to someone browsing the code.

I'd like your opinion on how I might name these symbols better.